askama/src/filters/json.rs

```diff
@@ -19,17 +19,74 @@ use serde_json::to_writer_pretty;
 /// or in apostrophes with the (optional) safe filter `'{{data|json|safe}}'`.
 /// In HTML texts the output of e.g. `<pre>{{data|json|safe}}</pre>` is safe, too.
 #[inline]
-pub fn json<S: Serialize>(s: S) -> Result<impl fmt::Display, Infallible> {
-    Ok(ToJson(s))
+pub fn json<S: Serialize, I: AsIndent>(
+    value: S,
+    indent: I,
+) -> Result<impl fmt::Display, Infallible> {
+    Ok(ToJson { value, indent })
```

Comment:
Is there reason not to store an `Option<'static str>` in `ToJson::indent`?

Reply:
Yes, the lifetimes wouldn't work.